### DISCUSSION TITLE
External network camera

### DISCUSSION THREAD

**Discussion by TheDescender** [fwestenberg/reolink_dev#583] [Q&A]
Apologies if this is a dumb question, but does this integration support connecting camera's in an external network? (through http or however)

I have a compound with Reolink cameras that are on another WAN that i'd like to connect back over to the HA environment on my home network. So I guess main question is, does this addon support this? and if not, do you know of any other ways to support this kind of interaction? (I looked at WebRTC, but haven't tried it yet)

**Comment by aclerici38** [comment]
I use an OpenVPN client add-on to remotely connect to an OpenVPN server on my other WAN. I've only had it running for a couple days so far but it's been working great

**Reply by TheDescender** [reply]
Yeah this is the solution I settled on.
